helper/testhelpers/pluginhelpers/pluginhelpers.go

```diff
@@ -131,12 +133,9 @@ func CompilePlugin(t testing.TB, typ consts.PluginType, pluginVersion string, pl
 	if _, err := os.Stat(pluginPath); os.IsNotExist(err) {
 		err = os.WriteFile(pluginPath, pluginBytes, 0o755)
 	}
-	if err != nil {
-		t.Fatal(err)
```

Comment:
Do we want to keep this one?

Reply:
Err was unassigned after my updates to the code, so it would always be nil! I removed `err`, and therefore the need for this check :)

Reply:
Oh, wait, maybe I got it mixed up, let me look...

Reply:
I think you're right that this might have mixed something up -- I refactored the code, and equivalent err checks are back!